m2cgen/assemblers/boosting.py

```diff
@@ -263,11 +314,11 @@ def _count_leaves(tree):
         return num_leaves
 
 
-def _split_trees_by_classes(trees, n_classes):
+def _split_estimators_by_classes(values, n_classes):
```

Comment:
I'm not sure, is it technically correct to call coefficients "estimators"?..

Reply:
Would `estimator_params` make more sense?

Reply:
I'd prefer something even more neutral like "values", but "estimator_params" is rather good too.